ml/notebooks/netdata_anomaly_detection_deepdive.ipynb

```diff
@@ -0,0 +1,1705 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">"source": [
    "We will collect and manipulate some data in this workbook with the [Pandas](https://pandas.pydata.org/) data analysis tool.\n",
"We will pull data from the [Netdata Agent REST API](https://learn.netdata.cloud/docs/agent/web/api) into a nice clean Pandas [DataFrame](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.html) where it will be easier to work with. \n",
"To install netdata-pandas](https://github.com/netdata/netdata-pandas) uncomment and run the next cell."
   ]
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netdata/netdata/pull/12313/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>